client/my-sites/stats/stats-countries/style.scss

```diff
@@ -4,34 +4,28 @@ $break-large-stats-countries: 1280px;
 
 // Countries
 .stats__module-list--traffic {
-	.stats__module-wrapper--countryviews .stats-module,
-	.list-countryviews {
-		display: grid;
-		gap: 0 24px;
-		grid-template-columns: 7fr 5fr;
-		grid-template-areas:
-			"map list"
-			"more more";
-
-		@media ( max-width: $break-large-stats-countries ) {
-			grid-template-columns: 100%;
-			grid-template-areas:
-				"map"
-				"list"
-				"more";
+	@media ( min-width: $break-large-stats-countries ) {
+		.stats__module-wrapper--countryviews .stats-module,
+		.list-countryviews {
+			display: flex;
+			flex-direction: row;
+			flex-wrap: wrap;
 		}
-	}
 
-	.list-countryviews {
-		.stats-card--hero {
-			grid-area: map;
-			padding-bottom: 0;
-		}
-		.stats-card--header-and-body {
-			grid-area: list;
-		}
-		.stats-card--footer {
-			grid-area: more;
+		.list-countryviews {
+			.stats-card--hero {
+				padding-bottom: 0;
```

Comment:
It looks like this isn't being properly applied at the moment.

Reply:
Thanks for catching this, @a8ck3n. I'll take a look at it.